src/export/meetbouten/types.py

```diff
@@ -97,6 +98,58 @@ def _to_geometry(value):
         .replace('.', ',')
 
 
+def _to_xcoord(value):
+    """Convert to xcoord
+
+    The geometry is translated to match the DIVA output format.
+
+    Example:
+
+        {
+            type: "Point",
+            coordinates: [
+                119411.7,
+                487201.6
+            ]
+        }
+        Output: 119411,7
+
+    :param value:
+    :return:
+    """
+    assert(type(value) is dict or value is None)
+    return _get_coord(value, 0)
```

Comment:
It is a liitle confusing that None results in ''.
Probably move the return '' outside _get_coord or add a parameter when_none='' or something like that